app/controllers/admin/users_controller.rb

```diff
@@ -49,6 +49,14 @@ def user_status
       redirect_to "/admin/users/#{@user.id}/edit"
     end
 
+    def export_data
```

Comment:
Just pinging @lightalloy because this method (and especially the worker it triggers) might be useful to know about for her upcoming GDPR work.